CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Package-specific changes not released in any SDK will be added here just before
 
 ### 📚 3rd party library updates
 
+- Updated `@react-native-community/netinfo` to `9.3.5`, `@react-native-picker/picker` to `2.4.8`, `@shopify/flash-list` to `1.3.1`, and `@shopify/react-native-skia` to `0.1.155` ([#19647](https://github.com/expo/expo/pull/19647) by [@brentvatne](https://github.com/brentvatne))
```

Comment:
nit: Cruzan told me to merge the changelog, so that users would be clear to see what versions we have, e.g.

```md
- Updated `react-native-screens` from `3.4.0` to `3.8.0`. ([#14330](https://github.com/expo/expo/pull/14330) by [@cruzach](https://github.com/cruzach)) ([#14544](https://github.com/expo/expo/pull/14544) by [@kudo](https://github.com/kudo))
```